core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -120,4 +239,12 @@ ReadableByteChannel get(PageId pageId, int pageOffset) throws IOException,
    * @return the number of pages stored
    */
   int size();
+
+  /**
+   * Loads page store from local storage and returns all page ids.
+   *
+   * @return collection of ids representing all pages loaded from disk
+   * @throws IOException if any error occurs
+   */
+  Collection<PageId> load() throws IOException;
```

Comment:
Both iterating files and rocksDB records require explicit call to `close()` method to release the resources. Returning an iterator hides this requirement and will lead to resource leak. Creating a copy of the list allow us to close the resource properly before this method returns. I would not concern too much about the memory consumption since we already store all page ids in meta store.